frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/copy_and_paste/custom_paste_command.dart

```diff
@@ -52,13 +55,16 @@ CommandShortcutEventHandler _pasteCommandHandler = (editorState) {
     Log.info('paste command: image: ${image?.$2?.length}');
 
     if (await editorState.pasteAppFlowySharePageLink(plainText)) {
```

Comment:
You can refactor the anonymous callback as a function. For example:

before
```
() async {} ()
```

after
```dart
Future<bool> doPaste() async {
  // if paste is handled , return true
  // return false
}
```

in command
```dart
{
  doPaste().then((r) if (r) { _handleDidPaste(context); } )
}

```